artifactory/commands/transferfiles/transfer_test.go

```diff
@@ -203,7 +203,8 @@ func TestUploadChunkAndPollUploads(t *testing.T) {
 
 // Sends chunk to upload, polls on chunk three times - once when it is still in progress, once after done received and once to notify back to the source.
 func uploadChunkAndPollTwice(t *testing.T, phaseBase *phaseBase, fileSample api.FileRepresentation) {
-	curThreads = 8
+	curChunkUploaderThreads = 8
+	curChunkBuilderThreads = 8
```

Comment:
Shouldn't we set the value of the above variables to `utils.DefaultThreads` instead of `8`?